src/test/java/trackitnus/logic/commands/module/AddModuleCommandIntegrationTest.java

```diff
@@ -44,10 +43,4 @@ public void execute_duplicateModule_throwsCommandException() {
         assertCommandFailure(new AddModuleCommand(moduleInList), model, Messages.MESSAGE_DUPLICATE_MODULE);
     }
 
-    @Test
-    public void execute_duplicateModuleCode_throwsCommandException() {
-        Module validModule = new Module(new Code("CS1101S"), new Name("Sample"));
-        assertCommandFailure(new AddModuleCommand(validModule), model, Messages.MESSAGE_DUPLICATE_MODULE);
-    }
-
```

Comment:
Hmm why did you remove this test case?

Reply:
because I have added a very similar testcase to line 41-45 
![Uploading image.png…]()

Reply:
because I have added a very similar testcase to line 41-45 
![image](https://user-images.githubusercontent.com/20136490/97837190-b2232380-1d18-11eb-8718-0c567171a652.png)